### PR TITLE
feat: Implement member key import key create route

### PR DESCRIPTION
**Pull request by levansuper**

### CODE REVIEW

File: src/services/schema/memberPublicKey.schema.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
+import { SERVICE_OID_REGEX } from './validaitons.js';
+
 export const MEMBER_PUBLIC_KEY_SCHEMA = {
   type: 'object',
 
   properties: {
     publicKey: { type: 'string' },
-    serviceOid: { type: 'string', pattern: '^(?:[0-9]+\\.)*[0-9]+$' },
+    serviceOid: { type: 'string', pattern: SERVICE_OID_REGEX },
```
Comment (levansuper) [inline src/services/schema/memberPublicKey.schema.ts:10]: was just a small change so also changed this

File: src/memberKeyImportToken.spec.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      expect(dbResult!.memberId).toStrictEqual(MEMBER_MONGO_ID);
+      expect(dbResult!.serviceOid).toStrictEqual(TEST_SERVICE_OID);
+      expect(mockLogging.logs).toContainEqual(
+        partialPinoLog('info', 'Member key import token created', {
+          id: meberKeyImportToken.result.id,
+        }),
+      );
+    });
+
+    test('Multiple tokens with same member id and service oid should be created', async () => {
```
Comment (levansuper) [inline src/memberKeyImportToken.spec.ts:48]: Not sure about the length of the description

**Comment by levansuper** [comment]
The business logic does not really any failure condition and that makes me think if I missed something. Maybe could pay attention to that

File: src/services/schema/validaitons.ts
Comment (gnarea) [inline src/services/schema/validaitons.ts:1]: File name has a typo. Should be `validations.ts` or `validation.ts` (better).

File: src/services/server.ts [hunk 3]
```diff
@@ -15,6 +16,7 @@ const ROUTES: FastifyPluginCallback<RouteOptions>[] = [
   orgRoutes,
   memberRoutes,
   memberPublicKey,
+  memberKeyImportToken,
```
Comment (gnarea) [inline src/services/server.ts:19]: Heads-up: Git won't report a merge conflict, but if #89 is merged first, things will break unless we manually move this to `member.routes.ts`.

File: src/memberKeyImportToken.spec.ts (hunk 2 above)
Comment (gnarea) [reply src/memberKeyImportToken.spec.ts:48]: Agreed. How about `Multiple tokens for same member and service should be allowed`?

**Review by gnarea — CHANGES_REQUESTED** [review]
LGTM, just some minor issues spotted.

**Comment by gnarea** [comment]
> The business logic does not really any failure condition and that makes me think if I missed something

I can't think of anything. The only things that could fail here related to the schema or authorisation, which are/will be handled separately.

File: src/services/schema/validaiton.ts
Comment (gnarea) [inline src/services/schema/validaiton.ts:1]: This file name is still wrong. Notice the typo.

File: src/services/server.ts
Comment (gnarea) [inline src/services/server.ts:1]: Merge gone wrong? This file shouldn't have changed.

**Review by gnarea — CHANGES_REQUESTED** [review]

**Review by gnarea — APPROVED** [review]
Thanks

**Pull request merged**